slack-market/api/endpoints/slack_endpoint.rb

```diff
@@ -0,0 +1,42 @@
+require 'pp'
+module Api
+  module Endpoints
+    class SlackEndpoint < Grape::API
```

Comment:
@dblock, I can definitely make that change.  Although one thought that comes to mind is building a more generalized SlackEndpoint class that responds to Slack Interactive messages/buttons.  I find that it may be easier for other contributors to recognize, while navigating the codebase.  Thoughts?